modules/Makefile.kubernetes

```diff
@@ -88,7 +88,8 @@ kubernetes\:replace:
 # (private) Update existing controller using rolling-update or create a new controller; do not notify datadog
 kubernetes\:deploy-controller:
 	@CURRENT_CONTROLLER_NAME=$$($(KUBECTL_CMD) get rc --selector="app=$(KUBERNETES_APP)" -o jsonpath="{.items[0].metadata.name}" 2>/dev/null); \
-  if [ -z "$$CURRENT_CONTROLLER_NAME" ] || [ $$? -ne 0 ]; then \
+	RESULT_CODE=$$?; \
+  if [ -z "$$CURRENT_CONTROLLER_NAME" ] || [ $$RESULT_CODE -ne 0 ]; then \
   	$(SELF) kubernetes\:create-controller; \
```

Comment:
Could you add an echo indicating we are creating a new controller because existing was not found. Similar to what is done below for rolling update